app/src/editor/canvas/components/Ports/Value/Text/index.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import React, { useState } from 'react'
+import cx from 'classnames'
+import EditableText from '$shared/components/EditableText'
+import { type CommonProps } from '..'
+import styles from './text.pcss'
+
+type Props = CommonProps & {}
+
+const Text = ({ disabled, onChange, value, ...props }: Props) => {
+    const [editing, setEditing] = useState(false)
+
+    return (
+        <EditableText
+            {...props}
+            className={cx(styles.root, {
```

Comment:
should allow a custom `className` to be supplied

Reply:
unless the intention is if supplying a custom `className` then the default classes will not apply at all?